db.go

```diff
@@ -528,6 +530,38 @@ func (db *DB) Close() error {
 	return merr.Err()
 }
 
+// ToggleCompactions toggles compactions and returns if compactions are on or not.
+func (db *DB) ToggleCompactions() bool {
```

Comment:
Mh, we should probably still leave those as separate `EnableCompactions`/`DisableCompactions`.

Especially as we have no way to check the current state, so we don't even know what a call of `Toggle` would get us into :) If we had a getter for the current state, practically, every call to Toggle should probably be preceded by that – so we are not really gaining anything.